tests/spectron/spectronWebDriver.js

```diff
@@ -0,0 +1,210 @@
+    async selectIMTab() {
+        var imTab = await this.getElementByXPath(ui.IM_TAB);
+        await imTab.click();
+    }
+    async  selectRoomTab() {
+        var roomTab = await this.getElementByXPath(ui.CHATROOM_TAB);
+        await roomTab.click();
+    }
+    async  addParticipant(username) {
+        await this.inputText(ui.ADD_PARTICIPANT_TEXT, username);
+        await this.sleep(5);
+        var el = await this.waitElementVisibleAndGet(ui.USERS_SUGGESTION_LIST);
+        await el.click();
+    }
+    async  clickDoneButton() {
+        var el = await this.getElementByXPath(ui.CREATE_IM_DONE_BTN);
+        await el.click();
+    }
+    async  clickStartChat() {
+        var el = await this.getElementByXPath(ui.START_CHAT);
+        await el.click();
+    }
+    async  createIM(username) {
+        await this.clickShowConversationCreationModal();
+        await this.clickStartChat();
+        await this.selectIMTab();
+        await this.addParticipant(username);
+        await this.clickDoneButton();
+    }
+
+    async  createMIM(usernames) {
+        await this.clickShowConversationCreationModal();
+        await this.clickStartChat();
+        await this.selectIMTab();
+        for (var i = 0; i < usernames.length; i++) {
+            await this.addParticipant(usernames[i]);
+        }
+        await this.clickDoneButton();
+    }
+    async  selectPublicRadioButton() {
+        var el = await this.waitElementVisibleAndGet(ui.PUBLIC_ROOM_RADIO_BTN);
+        await el.click();
+    }
+    async  selectPrivateRadioButton() {
+        var el = await this.waitElementVisibleAndGet(ui.PRIVATE_ROOM_RADIO_BTN);
+        await el.click();
+    }
+
+
+    async  clickLeftNavItem(name) {
+        xpath = ui.LEFT_NAV_SINGLE_ITEM.replace("$$", name);
+        var el = await this.waitElementVisibleAndGet(xpath);
+        await el.click();
+    }
+    async  createRoom(usernames, name, description, type) {
+        await this.clickShowConversationCreationModal();
+        await this.clickStartChat();
+        await this.selectRoomTab();
+        await this.inputText(ui.CHATROOM_NAME_TEXT, name);
+        await this.inputText(ui.CHATROOM_DESCR_TEXT, description);
+        if (type === specconst.TYPE_ROOM.private) {
+            await this.selectPrivateRadioButton();
+        }
+        if (type === specconst.TYPE_ROOM.public) {
+            await this.selectPublicRadioButton();
+        }
+        for (var i = 0; i < usernames.length; i++) {
+            await this.addParticipant(usernames[i]);
+        }
+        await this.clickDoneButton();
+    }
+    async  initDriver() {
+        return this.d.then(_d => {
+            this.driver = _d
+        })
+    }
+    async  startDriver() {
+        await this.driver
+            .manage()
+            .window()
+            .setPosition(0, 0);
+        var size = await await this.driver
+            .manage()
+            .window().getSize();
+        await this.driver.get(process.env.TESTED_HOST);
```

Comment:
let's use DRY principle here. move `process.env.TESTED_HOST` to a local variable called `TESTED_HOST` and apply it across the file.